src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -976,6 +1022,8 @@ public String toString() {
                 .append("EUt", EUt)
                 .append("hidden", hidden)
                 .append("cleanroom", getCleanroom())
+                .append("dimensions", getDimensionIDs().toString())
+                .append("dimensions_b", getBlockedDimensionIDs().toString())
```

Comment:
i feel like this should be renaimed to somthing more descriptive like `blocked_dimentions`